contractdata/CARPATHIAN/_CARPATHIAN_MASTERY.json

```diff
@@ -0,0 +1,192 @@
+[
+    {
+        "CompletionData": {
+            "Level": 1,
+            "MaxLevel": 5,
+            "XP": 0,
+            "Completion": 0.0,
+            "XpLeft": 6000,
+            "Id": "LOCATION_PARENT_TRAPPED",
+            "SubLocationId": "LOCATION_PARENT_TRAPPED",
+            "HideProgression": false,
+            "IsLocationProgression": true,
+            "Name": null
+        },
+        "Drops": [
+            {
+                "IsLevelMarker": false,
+                "Unlockable": {
+                    "Properties": {
+                        "Location": "LOCATION_PARENT_TRAPPED",
+                        "RepositoryId": "f21b1a1a-bec9-4a06-9a4a-d615e9721e3a",
+                        "UnlockOrder": 1,
+                        "LoadoutSettings": {
+                            "GearSlotsEnabledCount": 0,
+                            "GearSlotsAllowContainers": false,
+                            "ConcealedWeaponSlotEnabled": false
+                        },
+                        "Equip": [
+                            "TOKEN_OUTFIT_TRAPPED_WOLVERINE_SUIT"
+                        ]
+                    },
+                    "Rarity": null,
+                    "DisplayNameLocKey": "UI_STARTING_LOCATION_TRAPPED_WOLVERINE_TRAIN_LAB_NAME",
+                    "Id": "STARTING_LOCATION_TRAPPED_WOLVERINE_TRAIN_LAB",
+                    "Type": "access",
+                    "Subtype": "startinglocation",
+                    "GameAsset": null,
+                    "ImageId": "",
+                    "RMTPrice": -1,
+                    "GamePrice": -1,
+                    "IsPurchasable": false,
+                    "IsPublished": true,
+                    "IsDroppable": false,
+                    "Capabilities": [],
+                    "Qualities": {},
```

Comment:
These are duplicated from `allunlockables` - we don't want to waste this much memory on duplicated data.

Reply:
Agree, but see my comment under Anthony's - I think we have to pick a lesser evil here. I'm all for de-deplication though.